teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -186,6 +186,7 @@ interface TestRunsLocator {
     fun forTest(testId: TestId): TestRunsLocator
     fun forProject(projectId: ProjectId): TestRunsLocator
     fun withStatus(testStatus: TestStatus): TestRunsLocator
+    fun withExpandInvocations(expandInvocations: Boolean) : TestRunsLocator
```

Comment:
What does this method mean? It isn't evident from its name. Could you please write KDoc for it?

Reply:
It add possibility to list all invocations of test in run.

Reply:
Sorry, I still don't get it. What does it mean "return all invocations"? As far as I can see even without this flag all test invocations satisfying other filters are returned. I tried running different queries on our TeamCity instance and didn't notice a difference with or without this flag.

And if the value is `false` by default one we need to have a boolean flag in the method? There seems to be no sense to call `withExpandInvocations(false)`.

Reply:
I removed useless param. 

![image](https://user-images.githubusercontent.com/6361016/86586747-b0269f00-bf88-11ea-83bb-ec8b2c6a577d.png)

This is the only documentataion about this field in teamcity rest api documentation. My use case is that I retry failed tests and I want to retrieve all invocation of tests. This option was the only one which allows me to retrieve all results of particular test.

Reply:
I asked TeamCity guys about this parameter. It indeed specifies how to describe results when some test was executed several times in same build. By default such runs are aggregated into a single value, `duration` property will be the sum of durations of individual runs, and `status` will be `SUCCESSFUL` if and only if all runs are successful. If `expandInvocations` is `true` individual runs of such tests are reported as separate entries. I think we should describe this in KDoc. Also I think we can invent a better name for it. What about `expandMultipleInvocations`?

Reply:
Sure, make sense to me.